haproxy/metadata.csv

```diff
@@ -185,8 +185,11 @@ haproxy.process.unstoppable.jobs,gauge,,,,Current number of active jobs that can
 haproxy.process.uptime.seconds,gauge,,,,How long ago this worker process was started (>= 2.4).,0,haproxy,
 haproxy.server.bytes.in.total,count,,byte,,Current total of incoming bytes.,0,haproxy,
 haproxy.server.bytes.out.total,count,,byte,,Current total of outgoing bytes.,0,haproxy,
+haproxy.server.check.code,gauge,,,,"layer5-7 code, if available of the last health check. (>= 2.4)",0,haproxy,
+haproxy.server.check.duration.seconds,gauge,,second,,Previously run health check duration (>= 2.4),0,haproxy,
 haproxy.server.check.failures.total,count,,,,Total number of failed check (Only counts checks failed when the server is up).,0,haproxy,
-haproxy.server.check.last.change.seconds,gauge,,,,Number of seconds since the last UP<->DOWN transition.,0,haproxy,
+haproxy.server.check.status,gauge,,,,"Status of last health check, if enabled. (>= 2.4)",0,haproxy,
+haproxy.server.check.last.change.seconds,gauge,,second,,Number of seconds since the last UP<->DOWN transition.,0,haproxy,
```

Comment:
```suggestion
haproxy.server.check.last.change.seconds,gauge,,second,,Number of seconds since the last UP<->DOWN transition. (>= 2.4),0,haproxy,
```